react-components/src/components/utils/Thumbnail.tsx

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import styled from 'styled-components';
+import React, { useMemo } from 'react';
+import { Icon } from '@cognite/cogs.js';
+
+export const Thumbnail = ({
+  imageUrl,
+  isLoading,
+}: {
+  imageUrl?: string;
+  isLoading: boolean;
+}) => {
+  const image = useMemo(() => {
+    if (imageUrl) {
+      return (
+        <div>
+          <img src={imageUrl} alt="thumbnail" />
+        </div>
```

Comment:
Have you tested the toolbar-setup with real data? I imagine it might be a bit hard to do it, but we need to know whether it works :sweat_smile: